source/guides/models/repositories.md

```diff
@@ -95,6 +95,7 @@ class BookRepository < Hanami::Repository
       .where(author_id: author.id)
       .order(:published_at)
       .limit(limit)
+      .to_a
   end
 end
```

Comment:
So, if we add `#to_a` here we need to add it [here](https://github.com/hanami/hanami.github.io/pull/346/files#diff-d35c518e353ad9c912f2e43ed7db26f6R73). WDYT?